### ISSUE TITLE
iiwa: Should add in actual vel + effort + est. accel limits; attempt sync. for variants

### ISSUE THREAD

**Issue by EricCousineau-TRI** [RobotLocomotion/drake#15613]
At present, IIWA14 URDFs (at least) do not appear to have actual hardware velocity and torque limits; instead, they're derived from 
https://github.com/IFL-CAMP/iiwa_stack/tree/2fa5bd5 (understandably)
Additionally, they do not have acceleration limits, which are useful for planning and control.

We have TODOs in Anzu to hoist these to Drake URDFs; this issue should be resolved by:
- [x] Incorporating velocity and effort limits from KUKA, w/ unittest consistency check (#16003)
- [x] Providing a reasonable estimate of acceleration limits (the actual achievable limits) (#16377)
- [x] Ideally, providing guidance on how to sync them against our different variants (xacro and hand-edited)
  - Guidance was unittesting what is common among the variants (#16003)
- [x] Eric: Should leverage these changes in Anzu (see `anzu#8347`)
- [ ] Use xacro support in Drake for as many variants as possible

We'll need to consider downstream user impact as well (e.g. simulations will change), and figure out mitigation (if it's worth it).

\cc @thduynguyen @calderpg-tri @siyuanfeng-tri @IanTheEngineer 

@azeey I'd like us to queue this up starting October.

**Comment by EricCousineau-TRI** [comment]
Marco saw some difference in precision between prior version of ROS1 xacro and Noetic xacro in #16003.

**Comment by marcoag** [comment]
After going all the way back the `kinetic` branch of `xacro` the generated `urdf` still differs from the original. Here is the diff: 
https://gist.github.com/marcoag/fb5216f34ead56e34c44fb462f904d61 .Trying to use previous versions throws problems of very old and not supported dependencies not found, making it very hard to run.

It seems that the [file was committed in 2016](https://github.com/RobotLocomotion/drake/commit/4e561f01e0b2cf0937e5a56796b43ffdb5e4402c) so it must have been generated with a `kinetic` version or before.

From [the files blame description](https://github.com/RobotLocomotion/drake/blame/master/manipulation/models/iiwa_description/urdf/iiwa14_polytope_collision.urdf) it seems that the files have been manually modified several times, making it impossible to have the generated files exactly match the current committed version. 

Also given that [this line](https://github.com/RobotLocomotion/drake/blob/7445fa973aa8571800cf4a6ccb547f47bb797c6e/manipulation/models/iiwa_description/urdf/iiwa14.xacro#L447) is supposed to generate something like: `<joint name="iiwa_tool0_joint" type="fixed">` and the committed file has `<joint name="tool0_joint" type="fixed">` tells me the file has not even been regenerated any time after being committed, even when the `xacro` file was modified.

The rest of the differences are reordering and precision (values are more precise now).

I guess this leaves us the following options:

1. **Remove the generated files**. As generated files (sames as a binary) keeping them in the repo will only take space and confuse people if they are not updated. In this case, the [README.md ](https://github.com/RobotLocomotion/drake/blob/master/manipulation/models/iiwa_description/README.md) should explain how to generate the `urdfs`.
2. **Manually modify the generated files** and update velocity and effort.
3. **Regenerate and commit** the new version of the generated files.

Suggestions?

**Comment by EricCousineau-TRI** [comment]
My suggestion is to first do (3) as a separate PR, but ideally using `melodic` (since that is what we're using in Anzu). We should then commit that artifact to show the changes isolated to just the toolchain.

If we see something in the files that are hand-edited, we should try to hoist that back into the xacro, or just fork the model.

At some point, we should teach Drake to use `xacro` proper (assuming it's not too intertwined in ROS dependencies). We have this in Anzu.

**Comment by calderpg-tri** [comment]
We have a number of different iiwa models, all nominally based on the same xacro model, but with significantly different collision geometries. I suspect that (2) is the only way to update them all, unless we refactor them all to share some base xacro?

**Comment by EricCousineau-TRI** [comment]
Aye. At minimum, we should perhaps identify where divergence happens. Hopefully, (3) as a first check can tell us what's close and what's far?

**Comment by jwnimmer-tri** [comment]
If memory serves, the comments in the folder & files about "Use xacro for this" are grossly misleading, and you are going to find in (3) that there are mountains of divergence.

Paying off that debt and fixing the comments / xacro is a fine project, but seems like a huge, time-sink distraction on the way to getting correctly limits typed into the model files.  Adding the limits by hand into all of the files in git seems super easy to implement and code review, and could merge quick, and thus unblock the progress we need here.  We can service the debt immediately after, if you think it's important.

**Comment by EricCousineau-TRI** [comment]
Aye, in that case, (2) sounds better.

**Comment by marcoag** [comment]
Just went ahead and added (2) to https://github.com/RobotLocomotion/drake/pull/16003. Are we still looking to teach Drake to use xacro proper? if so I assume that should be done on a different PR, is it?

**Comment by marcoag** [comment]
@EricCousineau-TRI I was looking to move forward and send a PR on task 2 of this issue but wanted to confirm: Are the values from https://odr.chalmers.se/bitstream/20.500.12380/256658/1/256658.pdf acceptable for now?

**Comment by EricCousineau-TRI** [comment]
Yup!

**Comment by EricCousineau-TRI** [comment]
Going to carve out "variant" stuff to separate issue.

**Comment by EricCousineau-TRI** [comment]
Closing this in lieu of #8946
